examples/chart/event-handler/templates/deployment.yaml

```diff
@@ -44,7 +44,7 @@ spec:
               value: "true"
           {{- if .Values.tls.existingCASecretName }}
             - name: SSL_CERT_FILE
-              value: /etc/teleport-tls-ca/ca.pem
+              value: /etc/teleport-tls-ca/{{ .Values.tls.existingCASecretKeyName | default "ca.pem" }}
```

Comment:
Setting defaults both in values.yaml an in the template is confusing. Can we only rely on the default from `values.yaml` instead of adding a hardcoded default in the template?